man/sysctl.d.xml

```diff
@@ -51,20 +54,20 @@ key.middle/part/with/dots.foo = 123
     first non-whitespace character is <literal>#</literal> or
     <literal>;</literal> are ignored.</para>
 
-    <para>Note that either <literal>/</literal> or
-    <literal>.</literal> may be used as separators within sysctl
-    variable names. If the first separator is a slash, remaining
-    slashes and dots are left intact. If the first separator is a dot,
-    dots and slashes are interchanged.
-    <literal>kernel.domainname=foo</literal> and
-    <literal>kernel/domainname=foo</literal> are equivalent and will
-    cause <literal>foo</literal> to be written to
+    <para>Note that either <literal>/</literal> or <literal>.</literal> may be used as separators within
+    sysctl variable names. If the first separator is a slash, remaining slashes and dots are left intact. If
+    the first separator is a dot, dots and slashes are interchanged.
+    <literal>kernel.domainname=foo</literal> and <literal>kernel/domainname=foo</literal> are equivalent and
+    will cause <literal>foo</literal> to be written to
     <filename>/proc/sys/kernel/domainname</filename>. Either
     <literal>net.ipv4.conf.enp3s0/200.forwarding</literal> or
-    <literal>net/ipv4/conf/enp3s0.200/forwarding</literal> may be used
-    to refer to
-    <filename>/proc/sys/net/ipv4/conf/enp3s0.200/forwarding</filename>.
-    </para>
+    <literal>net/ipv4/conf/enp3s0.200/forwarding</literal> may be used to refer to
+    <filename>/proc/sys/net/ipv4/conf/enp3s0.200/forwarding</filename>. A glob
+    <citerefentry><refentrytitle>glob</refentrytitle><manvolnum>7</manvolnum></citerefentry> pattern may be
+    used to write the same value to all matching keys. Keys for which an explicit pattern exists will will be
```

Comment:
'will will'

Reply:
Thanks for the review. Pushed with just "will will" changed. The tests passed previously, so I'll just merge without waiting for them to run again.